packages/dd-trace/src/util.js

```diff
@@ -87,12 +87,83 @@ function generatePointerHash (components) {
   return hash.substring(0, 32)
 }
 
+/**
+ * Encodes a DynamoDB attribute value to Buffer for span pointer hashing.
+ * @param {Object} valueObject - DynamoDB value in AWS format ({ S: string } or { N: string } or { B: Buffer })
+ * @returns {Buffer} Encoded value as Buffer, or empty Buffer if invalid input.
+ *
+ * @example
+ * encodeValue({ S: "user123" }) -> Buffer("user123")
+ * encodeValue({ N: "42" }) -> Buffer("42")
+ * encodeValue({ B: Buffer([1, 2, 3]) }) -> Buffer([1, 2, 3])
+ */
+function encodeValue (valueObject) {
```

Comment:
I think alot of this util code is specific to the aws integration so I think it would make more sense to introduce a util.js file somewhere in packages/datadog-plugin-aws-sdk/src instead

Reply:
Agreed, thanks for the suggestion